### PR TITLE
help: Add /help command

### LINKED ISSUE #8: /help messages are wrong
Each module should probably present its own information using `/commands` or `/help` (maybe both?).

The messages must be fixed, they are currently broken.

The base scripts should also mention their own functions, such as `/rules`.

### PR DESCRIPTION
**Pull request by dracc**
Closes #8

Every module now outputs a list of their available commands.
Commands that require admin privilege are only shown for admins in a slightly brighter color.

### CODE REVIEW

File: scripts/module/debug.nut [hunk 1]
```diff
@@ -9,10 +9,10 @@ function onPlayerCommand( player, cmd, text ) {
 
     collectgarbage();
     ReloadScripts();
-  }
-
-  if (cmd.tolower() == "pos") {
+  } else if (cmd.tolower() == "pos") {
     print(player.Pos);
+  } else if (cmd.tolower() == "help") {
```
Comment (JayFoxRox) [inline scripts/module/debug.nut]: While you are at it, can you move the `.tolower()` like it's done in other scripts?

File: scripts/module/teleporter.nut [hunk 2]
```diff
@@ -36,6 +36,8 @@ function onPlayerCommand( player, cmd, text ) {
     } else {
       MessagePlayer("Usage: /" + cmd + " <string>", player);
     }
+  } else if (cmd == "help") {
+    MessagePlayer("[#333380]/t", player);
```
Comment (JayFoxRox) [inline scripts/module/teleporter.nut]: For these short forms we should probably use something like `/t[eleport]` and accept both commands in the command handler. Can be done in a separate PR.

File: scripts/base/freeroam.nut [hunk 3]
```diff
@@ -31,12 +31,13 @@ function onPlayerJoin( player ) {
...
   cmd = cmd.tolower();
   switch ( cmd ) {
-    case "rules": {
-      Message( "No disturbing other's play. Kids only." );
-      break;
-    }
-    default: {
-      break;
-    }
+  case "rules":
```
Comment (JayFoxRox) [inline scripts/base/freeroam.nut]: We should probably switch this to `if (cmd == "rules")` like in other scripts, or vice-versa. Basically just make it uniform :P

Can alternatively be done in a separate PR.

File: scripts/module/debug.nut [hunk 4]
```diff
@@ -3,15 +3,16 @@ function onScriptLoad() {
 }
 
 function onPlayerCommand( player, cmd, text ) {
-  if (cmd.tolower() == "r") {
+  cmd = cmd.tolower();
+  if (cmd == "r") {
```
Comment (JayFoxRox) [inline scripts/module/debug.nut]: r[eload]

File: scripts/module/admin.nut [hunk 5]
```diff
@@ -23,17 +23,12 @@ function GetTok(string, separator, n, ...) {
 
 function onPlayerCommand( player, cmd, text ) {
 
-  local lcmd = cmd.tolower();
+  cmd = cmd.tolower();
 
-  switch ( lcmd ) {
-  //FIXME: Top security brah
-  case "secret":
+  if (cmd == "secret") {
```
Comment (JayFoxRox) [inline scripts/module/admin.nut]: `//FIXME: Top security brah` is kind of important :P

File: scripts/module/emote.nut [hunk 6]
```diff
@@ -86,5 +86,7 @@ function onPlayerCommand(player, cmd, text) {
     }
     MessagePlayer(message, player);
     return;
+  } else if (cmd == "help") {
+    MessagePlayer("[#333380]/e", player);
```
Comment (JayFoxRox) [inline scripts/module/emote.nut]: e[mote]

**Comment by dracc** [comment]
I believe this to be ready for merge.

**Comment by dracc** [comment]
Fixed and working. Colors are ugly but readable.
